### PR TITLE
Merge two DataTypeUtil classes

### PR DESCRIPTION
**Pull request by jackylk**
There are two DataTypeUtil in carbon-core module, merging them into 1 class.

### CODE REVIEW

File: core/src/main/java/org/carbondata/scan/collector/impl/ListBasedResultCollector.java [hunk 1]
```diff
@@ -161,7 +160,7 @@ private Object getMeasureData(MeasureColumnDataChunk dataChunk, int index, DataT
         default:
           msrVal = dataChunk.getMeasureDataHolder().getReadableDoubleValueByIndex(index);
       }
-      return DataTypeUtil.getMeasureDataBasedOnDataType(msrVal, dataType);
+      return org.carbondata.core.util.DataTypeUtil.getMeasureDataBasedOnDataType(msrVal, dataType);
```
Comment (ravipesala) [inline core/src/main/java/org/carbondata/scan/collector/impl/ListBasedResultCollector.java]: Why not import `DataTypeUtil`? Please import in all other places also.

Comment (jackylk) [inline core/src/main/java/org/carbondata/scan/collector/impl/ListBasedResultCollector.java]: will fix it. It is generated by IDE refactory function

**Comment by ravipesala** [comment]
@jackylk can you raise these refactoring PR's on new apache git? we already migrated to apache git so all new PR can be raised to new git.

**Comment by jackylk** [comment]
@ravipesala I think the new repo is not ready, we still need to use this repo for some days

**Pull request closed without merging**